supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -247,19 +248,28 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
 
   private def modifyCommandForDocker(jobCmd: Try[String], localizedInputs: CallInputs): Try[String] = {
     Try {
-      val inputFiles = localizedInputs.filter { case (k,v) => v.wdlType.equals(WdlFileType) }
-      val dockerInputDataVol: Seq[String] = inputFiles.values.map { value =>
-        val limit = value.valueString.lastIndexOf("/")
-        value.valueString.substring(0, limit)
+      val inputFiles = localizedInputs.filter {
+        case (k, v) => v.wdlType == WdlFileType || v.wdlType == WdlArrayType(WdlFileType)
+      }
+      val dockerInputDataVol: Seq[String] = inputFiles.values.flatMap {
+        case file if file.wdlType == WdlFileType =>
+          val limit = file.valueString.lastIndexOf("/")
```

Comment:
I am not sure but I think it's quite possible to have folders as inputs in the `localizedInputs`, no? Would this work in that case? (I believe the parent folder would be mounted in that case). Either way, a comment would be good here.